### PR TITLE
nooo moore songname-null!

### PR DESCRIPTION
**Pull request by mayo78**
fixes this bug
image

### CODE REVIEW

**Review by stilic — APPROVED** [review]

**Review by TheKitBoi — APPROVED** [review]

**Review by megasssdev — APPROVED** [review]

**Review by MAJigsaw77 — APPROVED** [review]

**Review by Raltyro — APPROVED** [review]

**Review by charlesisfeline — APPROVED** [review]

**Review by steviegt6 — CHANGES_REQUESTED** [review]
The actual implementation looks fine, but your PR shouldn't change newline formatting. Should go in a different PR.

**Review by lunacynne — APPROVED** [review]

**Review by DM-kun — APPROVED** [review]

File: source/AchievementsMenuState.hx [hunk 1]
```diff
@@ -44,7 +44,7 @@ class AchievementsMenuState extends MusicBeatState
 
 		Achievements.loadAchievements();
 		for (i in 0...Achievements.achievementsStuff.length) {
-			if(!Achievements.achievementsStuff[i][4] || Achievements.achievementsMap.exists(Achievements.achievementsStuff[i][2])) {
```
Comment (Steviegt6) [inline source/AchievementsMenuState.hx:47]: This should go in a different PR.

**Review by steviegt6 — CHANGES_REQUESTED** [review]

**Review by StellaBiscuit — APPROVED** [review]
Finally someone added the check in case it's null, thank god

**Review by HeliProGamer — APPROVED** [review]

**Comment by Theoyeah** [comment]
rip mayo is banned

**Review by Wither362 — APPROVED** [review]
I love this!

I approve this!

**Pull request closed without merging**